client-app/src/examples/contact/mobile/AppComponent.ts

```diff
@@ -0,0 +1,22 @@
+import {hoistCmp, uses} from '@xh/hoist/core';
+import {appBar} from '@xh/hoist/mobile/cmp/header';
+import {panel} from '@xh/hoist/mobile/cmp/panel';
+import {Icon} from '@xh/hoist/icon';
+import {AppModel} from '../AppModel';
+import {contactsPage} from './DirectoryPanel';
```

Comment:
I've said elsewhere, but renaming on import is confusing - lets keep import names, file names and class names consistent throughout